community-operators/resource-locker-operator/0.1.0/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=resource-locker-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=alpha
+LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
+
+COPY 0.1.0/manifests /manifests/
+COPY 0.1.0/metadata /metadata/
```

Comment:
This Dockerfile will be used to build the bundle image from current dir, so the copy shoud be

```
COPY manifests /manifests/
COPY metadata /metadata/
```

also need update all other version Dockerfile for this, no need add bundle.Dockerfile directly under your operator dir.